### ISSUE TITLE
ESLint: 2.10.0 - Parsing error: Unexpected token =

### ISSUE THREAD

**Issue by moroshko** [babel/babel-eslint#312]
I'm using:

    eslint: 2.10.0
    babel-eslint: 6.0.4

and getting the following error:
> Parsing error: Unexpected token =

which points to class properties:

    export default class Autowhatever extends Component {
      static propTypes = {    // <--- error points to this line
    ...

I expected `babel-eslint` to support class properties out of the box.

What am I missing?

Here is my eslint config:
```json
{
  "env": {
    "es6": true,
    "node": true,
    "browser": true
  },
  "parser": "babel-eslint",
  "parserOptions": {
    "sourceType": "module"
  }
}
```

**Comment by moroshko** [comment]
Probably related: https://github.com/eslint/eslint/issues/6158

**Comment by stoikerty** [comment]
I get the same error. It throws the error on windows but not osx.

**Comment by broberson** [comment]
I get this error on osx. I don't see how this is related to eslint/eslint#6158 at all.

**Comment by hzoo** [comment]
It could be the same error as https://github.com/eslint/eslint/issues/6158 in that 2.10 currently makes it so ESLint always uses the default parser and not babel-eslint. Try 2.9 and if it works then you should pin to 2.9 until a patch is released.

**Comment by stoikerty** [comment]
I get this error on `eslint` 2.7, 2.9 and 2.10 on windows, using either `babel-eslint` 5.0.4 or 6.0.4.
On OSX it seems to work for 2.7 with babel-eslint 5.0.4 for me.

**Comment by hzoo** [comment]
Can someone check 2.10.1 which fixes the parser issue?

**Comment by kevhuang** [comment]
I updated ESLint to 2.10.1 and this issue went away for me.

**Comment by oliviertassinari** [comment]
Same here, it fixed 👍 .

**Comment by hzoo** [comment]
Cool. Yep, it basically turned off babel-eslint for everyone. If you don't don't get unexpected token there would of been an issue (or you don't need babel-eslint haha)

**Comment by perrin4869** [comment]
For some reason the error hasn't gone away for me yet: https://travis-ci.org/perrin4869/react-onenterkeydown/jobs/130315342

Anyone got a clue why? https://github.com/perrin4869/react-onenterkeydown/

**Comment by stoikerty** [comment]
I'm still seeing the error. I'm using `eslint-loader` which could have to do with it, it looks like [more people](https://github.com/MoOx/eslint-loader/issues/92) are seeing the issue as well.

**Comment by stoikerty** [comment]
I've gone back and forth, uninstalled everything concerning eslint locally and globally, then went back and tried installing `2.9.0` locally which @hzoo reported should work. I'm glad that that version works for me on windows after all. I must've tried too many things at once when I was testing different versions.
I'll just use `2.9.0` until a better patch is released. Thanks @hzoo

**Comment by xgqfrms-gildata** [comment]
# `"parser": "babel-eslint"` is OK!

```js
{
    "parser": "babel-eslint",
    "parserOptions": {
        "ecmaVersion": 6,
        "sourceType": "module",
        "ecmaFeatures": {
            "jsx": true,
            "modules": true,
            "experimentalObjectRestSpread": true
        }
    },
    "plugins": [
        "react"
    ],
    "extends": ["eslint:recommended", "plugin:react/recommended"],
    "rules": {
        "comma-dangle": 0,
        "react/jsx-uses-vars": 1,
        "react/display-name": 1,
        "no-unused-vars": "warn",
        "no-console": 1,
        "no-unexpected-multiline": "warn"
    },
    "settings": {
        "react": {
            "pragma": "React",
            "version": "15.6.1"
        }
    }
}

```
